changelog/bugfixes/2024-02-13-fix-kmod-static-nodes-creation.md

```diff
@@ -0,0 +1 @@
+- Resolved kmod static nodes creation in bootengine ([bootengine#85](https://github.com/flatcar/bootengine/pull/85))
```

Comment:
Maybe mention kubevirt here for users to understand?

Reply:
Added a new line below mentioning kubevirt, also added the same line into the commit message for easier search

Reply:
Thanks, just FYI: The order may change in the release notes but I think it's ok in this case (I normally do only a single long line for that reason)